Packs/CommonScripts/Scripts/StringSimilarity/StringSimilarity_test.py

```diff
@@ -20,10 +20,13 @@
         "StringB": "test",
         "SimilarityScore": 0.7272727272727273,
     }),
+    ("foo", "bar", 0.1, {}),
 ])
 def test_string_similarity(first_string, second_string, similarity_threshold, expected_result, mocker):
     try:
         result = stringSimilarity(first_string, second_string, similarity_threshold)
+        print(f'{result.outputs=}')
```

Comment:
Why print the results?

Reply:
Left there by mistake. Used for debugging. Removed.